src/kbc_modules/sample_kbc/mod.rs

```diff
@@ -11,18 +11,24 @@ use anyhow::*;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
+const HARDCODED_KEY: &'static [u8] = &[
+    217, 155, 119, 5, 176, 186, 122, 22, 130, 149, 179, 163, 54, 114, 112, 176, 221, 155, 55, 27,
+    245, 20, 202, 139, 155, 167, 240, 163, 55, 17, 218, 234,
+];
+
 // KBS specific packet
 #[derive(Serialize, Deserialize, Debug)]
 pub struct AnnotationPacket {
-    // The access information of KBS is passed to KBC module through annotation.
-    // key_url is used as an example here.
-    pub key_url: String,
-    pub wrapped_key: Vec<u8>,
+    // This is an example annotation packet format.
+    pub kid: String,
+    pub wrapped_data: Vec<u8>,
+    pub iv: Vec<u8>,
```

Comment:
The commit message is "Modify sample KBC to support CCv0 PoC". It wouldn't be entirely clear to me which changes are important to this. Is it these fields?